File: tests/fixtures/python/simple-loop.py
"""Simple loop for basic stepping and variable inspection."""


def sum_range(n: int) -> int:
    total = 0
    for i in range(n):
        total += i
    return total


if __name__ == "__main__":
    result = sum_range(10)
    print(f"Sum: {result}")

File: tests/fixtures/csharp/SimpleLoop.cs
class SimpleLoop
{
    static int SumRange(int n)
    {
        int total = 0;
        for (int i = 0; i < n; i++)
        {
            total += i;
        }
        return total;
    }

    static void Main()
    {
        int result = SumRange(10);
        System.Console.WriteLine($"Sum: {result}");
    }
}
